halyard-config/src/main/java/com/netflix/spinnaker/halyard/model/v1/providers/Provider.java

```diff
@@ -22,7 +22,7 @@
 import java.util.List;
 
 @Data
-public class Provider<T extends Account> {
+public class Provider<T extends Account> implements Cloneable {
```

Comment:
`T extends Account` can also just be `Account`, since everything of the form `class _ extends Account` will also be an `Account`.